docs/developer-guide/parallel-execution.md

```diff
@@ -0,0 +1,19 @@
+---
+description: StreamingFast Substreams parallel execution
+---
+
+# Parallel execution
+
+Parallel execution is the process of a Substreams module's code executing multiple segments of blockchain data simultaneously. Substreams modules are executed in parallel, rapidly producing data for consumption in end-user applications. Parallel Execution enables Substreams' highly effecient blockchain data processing capabilities.
```

Comment:
```suggestion
Parallel execution is the process of a Substreams module's code executing multiple segments of blockchain data simultaneously. Substreams modules can be executed in parallel, rapidly producing data for consumption in end-user applications. Parallel Execution enables Substreams' highly efficient blockchain data processing capabilities.
```

Reply:
These updates have been made and will be available for review in the next commit/push.